WcaOnRails/spec/mailers/job_failure_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe JobFailureMailer, type: :mailer do
+  describe "notify_admin_of_job_failure" do
+    let(:job) { Delayed::Job.new(id: 42) }
+    let(:exception) do
+      begin
+        raise "error!"
+      rescue Exception => e
+        return e
+      end
+    end
+    let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, exception) }
```

Comment:
I think this works:
`let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, RuntimeError.new("error!")) }`